common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -86,6 +86,47 @@
 source, template_path = Loader(engine).load_template_source(path)
 %>${source | n, decode.utf8}</%def>
 
+<%def name="studiofrontend(page, lang='en')">
+    <%doc>
+        Loads a studio-frontend page, with the necessary context. Context is expected
+        as a dictionary in the body of this tag.
+
+        Dev note: we could also add the locale-injection script in this block
+            -use a better default than the hardcoded 'en'. There should be a setting or something?
+            -lookup (webpack exported) locale-injection script using lang as key
+            -include it as the first script in this block
+    </%doc>
+    <%
+        from django.template import Template, Context
+        from webpack_loader.exceptions import WebpackLoaderBadStatsError
+        import json
+
+        def _erics_hacky_helper(input_dict):
```

Comment:
The purpose of this function is to get our dict dumped as `{key1: "value1", key2: "value2"}` instead of `{u"key1": u"value1", u"key2", u"value2"}`